src/App.js

```diff
@@ -1,10 +1,32 @@
 import React from 'react';
-import './App.css';
+import withModal from './hoc/withModal';
+import Header from './components/Header/index';
+import withStoriesContainer from './hoc/withStoriesContainer';
+import { HashRouter as Router, Route } from 'react-router-dom';
+import FullStoryView from './components/Stories/FullStoryView';
+import StoriesContainer from './components/Stories/StoriesContainer';
```

Comment:
folderName should always be camelcase like fooBar.

Only component file name can be uppercase i.e FooBar.js